RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -407,10 +409,64 @@ int remainingEmbers()
 
 void auto_buyFromSeptEmberStore()
 {
+	if(!auto_haveSeptEmberCenser())
+	{
+		return;
+	}
 	if(remainingEmbers() == 0)
 	{
 		return;
 	}
+
+	// mouthwash for leveling
+	item mouthwash = $item[Mmm-brr! brand mouthwash];
+	boolean disregard_karma = get_property("auto_disregardInstantKarma").to_boolean();
+	auto_openMcLargeHugeSkis(); // make sure our skis are open for cold res
+	for (int imw = 0 ; imw < 3 ; imw++) // We can use up to 3 mouthwash
+	{
+		// If we have at least 4 embers remaining, don't overlevel, they can be used for something else
+		boolean happy_to_overlevel = disregard_karma && remainingEmbers() < 4;
+		boolean want_to_mouthwash_level = (my_level() < 13 || happy_to_overlevel) && my_level()<15;
+		// Even disregarding karma, never level above 15 using mouthwash as a sanity limit
+		want_to_mouthwash_level = want_to_mouthwash_level && my_level()<15;
+		if (remainingEmbers() >= 2 && want_to_mouthwash_level)
+		{
+			// get as much cold res as possible
+			int [element] resGoal;
+			resGoal[$element[cold]] = 100;
+			// get cold res. Use noob cave as generic place holder
+			
+			// get 1 bembershoot to support mouthwash leveling or general quest help
+			item bember = $item[bembershoot];
+			if (remainingEmbers() % 2 == 1 && !possessEquipment(bember) && auto_is_valid(bember))
+			{
+				buy($coinmaster[Sept-Ember Censer], 1, bember);
+			}
+			
+			provideResistances(resGoal, $location[noob cave], true);
+			equipMaximizedGear();
+			
+			// We could have left-hand if our off-hand is strong enough
+			if (numeric_modifier(equipped_item($slot[off-hand]),$modifier[cold resistance]) > 2.9)
```

Comment:
currently this won't work for mchugelarge, as the bonus is implemented as an outfit bonus. As McHugeLarge varies with the number of equipped items you'll need to special-case it: check for the pole being equipped and also `numeric_modifier($modifier[mchugelarge]) > 4 `

Reply:
I'm just checking for the off-hand being equipped, it pretty much can't go wrong since you get all the items at the same time.